server/pbenchinacan/container-build.sh

```diff
@@ -118,12 +116,13 @@ buildah run $container chown --recursive pbench:pbench /srv/pbench
 # buildah run $container semanage fcontext -a -t httpd_sys_content_t /srv/pbench/public_html/static
 # buildah run $container restorecon -v -r /srv/pbench/archive /srv/pbench/public_html
 
-buildah run $container crontab -u pbench ${SERVER_LIB}/crontab/crontab
-
 sed -e "s/localhost/${HOSTNAME_F}/" ${PBINC_SERVER}/lib/config/pbench.httpd.conf >/tmp/pbench.conf.${$}
 buildah copy --chown root:root --chmod 0644 $container \
     /tmp/pbench.conf.${$} /etc/httpd/conf.d/pbench.conf
 rm /tmp/pbench.conf.${$}
+# We need to ensure HTTPD listens on port 8080 so container can be optionally
+# run with host networking from a non-root user.
+buildah run $container bash -c "sed -i -e 's/^Listen 80$/Listen 8080/' /etc/httpd/conf/httpd.conf"
```

Comment:
At line 119 we are modifying (the file which _will_ be) `http.conf`, and then we modify it again at line 125.  At line 119 we modify it outside the container, but at line 125 we have to reach inside the container to modify it.

It would be better to make both modifications outside the container; and it would be better still, since they both use `sed`, to do it in a single command.  (If, that is, we don't decide to drop the second modification.)

Reply:
I have pulled the port change out of this PR so that we can work on the implications of that separate from these changes.  See PR #3108.